src/vs/workbench/contrib/chat/browser/actions/quickQuestionActions/quickQuestionAction.ts

```diff
@@ -25,6 +27,7 @@ export interface IQuickQuestionMode {
 	run(accessor: ServicesAccessor, query: string): void;
 }
 
+// TODO: This should be registered per chat-provider probably.
```

Comment:
Don't do a bunch of work on this, I was probably going to flip it around to make the registered chat view a single global thing